sandbox/katas/Greed.01/Greed/GreedGame.cs

```diff
@@ -0,0 +1,64 @@
+public class GreedGame
+{
+    public GreedGame()
+    {
+    }
+
+    public List<int> ReturnDices()
+    {
+        int diceNumber = 5;
+        int diceCounter = 1;
+        var diceList = new List<int>();
+        for (int i = 1; diceCounter <= diceNumber; diceCounter++)
```

Comment:
i tu máš zbytočne, veď pracuješ len s premennou dice counter. Takže by to mohlo kľudne vyzerať takto: 
for (int diceCounter = 1; diceCounter <= diceNumber; diceCounter++)